### ISSUE TITLE
use lstm

### ISSUE THREAD

**Issue by ancileddu** [tensorflow/workshops#31]
Hello, thank you very much for your work. I need to use a level of lstm..can you help me? I tried to insert
model.add(LSTM(activation='softmax', units=300, recurrent_activation='hard_sigmoid', return_sequences=True)) after the model.add(Dropout(0.5)) but i've the error: Input 0 is incompatible with layer lstm_1: expected ndim=3, found ndim=2
Sorry for my bad english. 
Thank you very much. Bye
